.gitignore

```diff
@@ -6,3 +6,4 @@
 tags
 /build/logs
 *.phar
+.DS_Store
```

Comment:
Please remove this. It doesn't belong to the project.

Reply:
Removed :D